tools/cr/brockit.py

```diff
@@ -940,7 +940,10 @@ def create_push_request(self, issue_url: str) -> bool:
             )
             return True
 
-        tag = f'[{upstream_branch}] ' if upstream_branch != 'master' else ''
+        # That's a naive way to think about this, but in general the uplift
+        # branches are upstreamed to 1.77.x, 1.78.x, etc.
+        is_uplift = upstream_branch.endswith('.x')
```

Comment:
Maybe we can make it a bit more bullet-proof by looking for `1\.\d+\.x` (not sure if that regex is legit in Python, but you get the idea), as long as we standarize the branch names to end like that?

Reply:
Wouldn't that break if we it gets flipped to `2.` though? Sorry, I'm not very good with regexes.

Reply:
Fixed as `r"^\d+\.\d+\.x$"`. I guess this is less naive.

Reply:
Yep, that should help a bit!